crates/nu-cli/src/commands/keybindings_get.rs

```diff
@@ -0,0 +1,354 @@
+use crossterm::event::{KeyCode, KeyEventKind, KeyModifiers, MouseEvent, MouseEventKind};
+use crossterm::terminal;
+use nu_engine::CallExt;
+use nu_protocol::ast::Call;
+use nu_protocol::engine::{Command, EngineState, Stack};
+use nu_protocol::{
+    Category, IntoPipelineData, PipelineData, ShellError, Signature, Span, SyntaxShape, Type, Value,
+};
+
+#[derive(Clone)]
+pub struct KeybindingsGet;
+
+impl Command for KeybindingsGet {
+    fn name(&self) -> &str {
+        "keybindings get"
+    }
+
+    fn signature(&self) -> Signature {
+        Signature::build(self.name())
+            .category(Category::Platform)
+            .named(
+                "types",
+                SyntaxShape::List(Box::new(SyntaxShape::String)),
+                "Listen for events of specified types only (can be one of: focus, key, mouse, paste, resize)",
+                Some('t'),
+            )
+            .input_output_types(vec![(
+                Type::Nothing,
+                Type::Record(vec![
+                    ("keycode".to_string(), Type::String),
+                    ("modifiers".to_string(), Type::List(Box::new(Type::String))),
+                ]),
+            )])
+    }
+
+    fn usage(&self) -> &str {
+        "Get keyboard event from user"
```

Comment:
I'm thinking `input listen` as a command name? It seems like it's what this does and it's not limited to keyboard events.
```suggestion
        "Listen for user interface event"
```

Reply:
Done